### ISSUE TITLE
Ssh problem 

### ISSUE THREAD

**Issue by nik9276** [rapid7/metasploitable3#573]
## Issue Description
ssh loading is infinitive
Running on Windows10 and vmware 
## Host System
* OS: Windows 2008 Server
* Packer Version: v1.8.1
* Vagrant Version: v2.2.19
* vmWare Version: 16.0.0

## Command Output
![image](https://user-images.githubusercontent.com/96139259/174498487-e471cc34-7f97-46f7-b212-e6a5b4a23b14.png)
it has been like this for at least 1 hour.
i had tried reading some of ticket closed but no one help me.
the command that i use is one of the "to build manually" and this is  --> packer build --only=vmware-iso ./packer/templates/windows_2008_r2.json

**Comment by jmartin-r7** [comment]
More details are need to track this down. I suspect the initial OS install configuration is failing to complete at some stage.  Watching the OS install may provide some hints as to what step in the `Autounattend.xml` configuration is failing to complete.  The OS initial prep needs also downloads some files from the internet during configuration, one of which is the step to install SSH.

**Comment by jetson02** [comment]
I've had this problem too in virtualbox, I'd started in bridged to ensure a net connection, and I could even able to ssh to the box manually, and browse the web and download. But I noticed the script tries to connect to 127.0.0.1, so I changed networking to NAT and the script picked up the SSH connection and continued,

However, then I get stuck further down the build at vagrant-ssh.ps1 & chocolatey.ps1 with long error messages (I've tried switching back to bridging before the builder runs the ps1 scripts, but it just freezes up and crashes, I suppose it severed the ssh connection at this point that must be required to run the vagrant & chocolatey powershell scripts.

maybe switching to NAT for the ssh connection will end up working for you. 

No matter what I do I can't get past vagrant-ssh.ps1 & chocolatey.ps1  and looking in the logs it appears that  "Dotnet 4.5""  & other download fails, but the openSSH download succeeds. I don't know what's going on.

I'd love to hear from anyone that can successfully run this build.
